### DISCUSSION TITLE
Couldn't notify the webapp that project was indexed

### DISCUSSION THREAD

**Discussion by sivabalan37** [oracle/opengrok#3970] [General]
**Describe the bug**
when execute the opengrok-indexer, i see this issue.

_Couldn't notify the webapp that project org.opengrok.indexer.configuration.Project@499ac77e was indexed: InboundJaxrsResponse{context=ClientResponse{method=PUT, uri=http://localhost:8080/comhem/api/v1/projects/source_code_1/indexed, status=500, reason=Internal Server Error}}_

```
PROJECT_NAME=comem
PROJECT_BASE=/home/spalanisamy/workspace/opengrok/comhem
sudo ~/.local/bin/opengrok-indexer \
            -J=-Djava.util.logging.config.file=$PROJECT_BASE/opengrok/etc/logging.properties \
              -a $PROJECT_BASE/opengrok/dist/lib/opengrok.jar -- \
                 -c /usr/local/bin/ctags  \
                    -s $PROJECT_BASE/opengrok/src  -d $PROJECT_BASE/opengrok/data/ -H -P -S -G \
                       -W $PROJECT_BASE/opengrok/etc/configuration.xml -U http://localhost:8080/$PROJECT_NAME

```
```
OS : 18.04.1-Ubuntu
opengrok: 1.7.23
tomcat: 10.0.13
 java --version
openjdk 11.0.11 2021-04-20
OpenJDK Runtime Environment (build 11.0.11+9-Ubuntu-0ubuntu2.18.04)
OpenJDK 64-Bit Server VM (build 11.0.11+9-Ubuntu-0ubuntu2.18.04, mixed mode, sharing)

```
when i access this via webpage, i see error as below
![image](https://user-images.githubusercontent.com/11088767/143441267-b6328517-3da2-4b9d-a704-d0d52bb0f6aa.png)

**Comment by vladak** [comment]
Does the source root path exist ? How did you deploy the web app ?

**Comment by sivabalan37** [comment]
i deploy as below
```
PROJECT_NAME=comem
PROJECT_BASE=/home/spalanisamy/workspace/opengrok/comhem
 ~/.local/bin/opengrok-deploy -c $PROJECT_BASE/opengrok/etc/configuration.xml  $PROJECT_BASE/opengrok/dist/lib/source.war  /usr/share/apache-tomcat/webapps/${PROJECT_NAME}.war
```

**Comment by vladak** [comment]
What is the value of `sourceRoot` in `$PROJECT_BASE/opengrok/etc/configuration.xml` ?

**Comment by sivabalan37** [comment]
`sourceRoot` value `/home/spalanisamy/workspace/opengrok/comhem/opengrok/src`
